kafka.go

```diff
@@ -27,6 +30,38 @@ func NewKafkaProducer(logger *log.Logger, stats *Stats, config *Config) (NozzleP
 	// TODO (tcnksm): Enable to configure more properties.
 	producerConfig := sarama.NewConfig()
 
+	if !config.Kafka.InsecurePlainText {
+		if len(config.Kafka.CACerts) == 0 {
+			return nil, errors.New("please specify at least one ca_certificate, else set insecure_use_plaintext")
+		}
+		if config.Kafka.ClientCert == "" {
+			return nil, errors.New("please specify client_certificate, else set insecure_use_plaintext")
```

Comment:
is mutual TLS required?

Reply:
I don't know if strictly required by Kafka or not - but I'm not sure it would be sensible not to. ie the client cert is used to authenticate the client to Kafka - so if not presented I guess at the least the connection is encrypted, but the server would still be wide open. (I may be missing a use-case?)